dll/win32/shell32/shlfileop.cpp

```diff
@@ -1752,7 +1776,15 @@ static void move_to_dir(FILE_OPERATION *op, const FILE_ENTRY *feFrom, const FILE
     PathCombineW(szDestPath, feTo->szFullPath, feFrom->szFilename);
 
     if (IsAttribFile(feFrom->attributes))
+#ifdef __REACTOS__
+    {
+        WCHAR szDestPath[MAX_PATH];
+        PathCombineW(szDestPath, feTo->szFullPath, feFrom->szFilename);
```

Comment:
Why isn't it possible to just keep what's inside szDestPath? Three lines above you have already a PathCombineW call that exactly does the very same thing (and it doesn't seem that feTo/feFrom strings have changed in-between).

Reply:
Fixed.